training/creating_dataset.py

```diff
@@ -118,9 +85,33 @@ def subcolumn_as_value(df: pd.DataFrame) -> pd.DataFrame:
     return subcolumn_as_value
 
 
+def load_data_with_nan_val(
+    column_name: str, nan_replace_val: int, original_column_name: str = "value"
+) -> callable:
```

Comment:
No po nazwie funkcji to bym się nie spodziewał czegoś takiego. Czemu to zwraca callable? Czemu po prostu nie można zrobić samej funkcji wewnętrznej i jej wprost użyć?